src/components/Home/StationModalContent.tsx

```diff
@@ -50,39 +58,167 @@ export default function StationModalContent({
         isOpen && getWeatherData();
     }, [isOpen]);
 
+    const transformDateToReadable = (inputDate: Date): string => {
```

Comment:
do you think we can rename this to getFormattedDate or formatDate?

Reply:
Yes, much better name than mine.